scene/animation/animation_node_state_machine.cpp

```diff
@@ -565,7 +565,7 @@ void AnimationNodeStateMachine::replace_node(const StringName &p_name, Ref<Anima
 	{
 		Ref<AnimationNode> node = states[p_name].node;
 		if (node.is_valid()) {
-			node->disconnect_compat("tree_changed", this, "_tree_changed");
+			node->disconnect("tree_changed", Callable(this, "_tree_changed"));
```

Comment:
You can use `callable_mp(this, &AnimationNodeStateMachine::_tree_changed)` here instead of relying on the method binding. Same for the other one.

Reply:
BTW this change probably fixed a bug, as `_tree_changed` is not bound in AnimationNodeStateMachine, so the previous code or the `Callable()` variant would have raised an error. I guess this code is from a PR which was authored before `callable_mp` changes, and merged after.